samples/nrf9160/mqtt_simple/src/main.c

```diff
@@ -10,6 +10,10 @@
 #include <string.h>
 #include <random/rand32.h>
 
+#if defined(CONFIG_BOARD_QEMU_X86) && !defined(CONFIG_BSD_LIBRARY)
```

Comment:
Let's include this file for all builds, and offer certificate provisioning functionality also for builds using BSDlib